regression-test/suites/nereids_syntax_p0/match.groovy

```diff
@@ -0,0 +1,178 @@
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_nereids_match_select") {
+    sql """
+        SET enable_nereids_planner=true
+    """
+
+    sql "SET enable_fallback_to_original_planner=false"
+
+    sql """
+        drop table if exists test_nereids_match_select;
+    """
+
+    sql """
+        CREATE TABLE `test_nereids_match_select`
+        (
+            `name`           varchar(50),
+            `age`            int NOT NULL,
+            `grade`          varchar(30) NOT NULL,
+            `fatherName`     varchar(50),
+            `matherName`     varchar(50),
+            `selfComment`    text,
+            INDEX name_idx(name) USING INVERTED PROPERTIES("parser"="english") COMMENT 'name index',
+            INDEX age_idx(age) USING INVERTED COMMENT 'age index',
+            INDEX grade_idx(grade) USING INVERTED PROPERTIES("parser"="none") COMMENT 'grade index',
+            INDEX fatherName_idx(fatherName) USING INVERTED PROPERTIES("parser"="english") COMMENT 'fatherName index',
+            INDEX matherName_idx(matherName) USING INVERTED PROPERTIES("parser"="english") COMMENT 'matherName index',
+            INDEX selfComment_idx(selfComment) USING INVERTED PROPERTIES("parser"="standard") COMMENT 'selfComment index'
+        ) ENGINE=OLAP
+        DUPLICATE KEY(`name`)
+        DISTRIBUTED BY HASH(`name`) BUCKETS 10
+        PROPERTIES (
+        "replication_allocation" = "tag.location.default: 1",
+        "in_memory" = "false",
+        "storage_format" = "V2"
+        );
+    """
+    
+    sql """ insert into test_nereids_match_select VALUES
+        ("zhang san", 10, "grade 5", "zhang yi", "chen san", "Class activists"),
+        ("zhang san yi", 11, "grade 5", "zhang yi", "chen san", "learn makes me happy"),
+        ("li si", 9, "grade 4", "li er", "wan jiu", "i just want go outside"),
+        ("san zhang", 10, "grade 5", "", "", ""),
+        ("li sisi", 11, "grade 6", "li ba", "li liuliu", "")
+    """
+    
+
+    order_qt_match_1 """
+        SELECT * FROM test_nereids_match_select WHERE name match 'zhang san';
+    """
+
+    order_qt_match_2 """
+        SELECT * FROM test_nereids_match_select WHERE name match 'zhang li';
+    """
+
+    order_qt_match_3 """
+        SELECT * FROM test_nereids_match_select WHERE name match 'zhang';
+    """
+
+    order_qt_match_4 """
+        SELECT * FROM test_nereids_match_select WHERE name match 'sisi';
+    """
+
+    order_qt_match_5 """
+        SELECT * FROM test_nereids_match_select WHERE name match 'zhang li' and grade = 'grade 5';
+    """
+
+    order_qt_match_6 """
+        SELECT fatherName, matherName FROM test_nereids_match_select WHERE name match 'zhang' or grade = 'grade 6';
+    """
+
+    order_qt_match_7 """
+        SELECT * FROM test_nereids_match_select WHERE name match 'zhang' and selfComment match 'happy';
+    """
+
```

Comment:
redundant blank line. if u want split different function test. the better way is add comment, not a redundant blank line

Reply:
fixed